src/core/seqner.rs

```diff
@@ -0,0 +1,268 @@
+use lazy_static::lazy_static;
+
+use crate::core::matter::{tables as matter, Matter};
+use crate::error::{err, Error, Result};
+
+#[derive(Debug, Clone, PartialEq)]
+pub struct Seqner {
+    raw: Vec<u8>,
+    code: String,
+    size: u32,
+}
+
+impl Default for Seqner {
+    fn default() -> Self {
+        Seqner { raw: vec![], code: matter::Codex::Salt_128.to_string(), size: 0 }
+    }
+}
+
+fn validate_code(code: &str) -> Result<()> {
+    lazy_static! {
+        static ref CODES: Vec<&'static str> = vec![matter::Codex::Salt_128,];
+    }
+
+    if !CODES.contains(&code) {
+        return err!(Error::UnexpectedCode(code.to_string()));
+    }
+
+    Ok(())
+}
```

Comment:
```suggestion
fn validate_code(code: &str) -> Result<()> {
//    const CODES: &[u8] = &[matter::Codex::Salt_128,];

//    if !CODES.contains(&code) {
//        return err!(Error::UnexpectedCode(code.to_string()));
//    }

    if code != matter::Codex::Salt_128 {
        return err!(Error::UnexpectedCode(code.to_string()));
    }

    Ok(())
}
```

Reply:
I realized:
1) we don't need lazy_static for these, see #99 
2) with only one comparison we should leave comments in case we need to add more codes, but do the comparison directly for performance

Reply:
See #97 for an example, in `Dater`